x11-misc/bumblebee/bumblebee-9999.ebuild

```diff
@@ -90,6 +88,8 @@ src_install() {
 	newenvd "${FILESDIR}"/${PN}.envd 99${PN}
 	systemd_dounit scripts/systemd/bumblebeed.service
 
+	local DOC_CONTENTS="In order to use Bumblebee, add your user to 'bumblebee' group.
+		You may need to setup your /etc/bumblebee/bumblebee.conf"
 	readme.gentoo_create_doc
 
 	default
```

Comment:
does `default` not work as the first command in `src_install`? This is the preferred style (unless it needs the dirs created by `newconfd` etc., which is unlikely)